recipes/centos_vertx/cccp.yml

```diff
@@ -0,0 +1 @@
+job-id: vertx
```

Comment:
Mmmh I'm not sure. I think we need one for every build job (i.e. one for every Dockerfile). @dharmit can you please confirm that?